redash/tasks/failure_report.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+from collections import Counter
+from flask import render_template
+from redash.tasks.general import send_mail
+from redash.worker import celery
+from redash import redis_connection, settings
+from redash.utils import json_dumps, json_loads, base_url
+
+
+@celery.task(name="redash.tasks.send_aggregated_errors")
+def send_aggregated_errors(email_address):
+    key = 'aggregated_failures:{}'.format(email_address)
```

Comment:
Using email address as a key is a really bad idea, considering that the same email address can be in multiple organizations. Why not use the user id?

Reply:
Yeah that makes sense. Fixed in 0bfe9f4.